server/services/Mailer.js

```diff
@@ -34,21 +32,18 @@ class Mailer extends helper.Mail {
   addRecipients() {
     const personalize = new helper.Personalization();
 
-    this.recipients.forEach(recipient => {
-      personalize.addTo(recipient);
-    });
+    this.recipients.forEach(recipient => personalize.addTo(recipient));
```

Comment:
For simple arrow functions like these, I find the single line version easier to read. I only use the multi-line version when I have to do multiple things in the function.